cli/command/service/logs.go

```diff
@@ -112,7 +112,7 @@ func runLogs(dockerCli *command.DockerCli, opts *logsOptions) error {
 			return err
```

Comment:
After you move the above code to the other noted spot, add a simple error check here for the result of `cli.TaskLogs`

```go
responseBody, err := cli.TaskLogs(ctx, opts.target, options)
if err != nil {
    return err
}
```